internal/js/modules/k6/browser/common/frame.go

```diff
@@ -1478,16 +1474,9 @@ func (f *Frame) selectOption(selector string, values sobek.Value, opts *FrameSel
 }
 
 // SetContent replaces the entire HTML document content.
-func (f *Frame) SetContent(html string, opts sobek.Value) error {
+func (f *Frame) SetContent(html string, _ *FrameSetContentOptions) error {
```

Comment:
Sounds like here (and maybe upstream) we can clean up the opts completely? :thinking:

Reply:
I don't know, for this reason, I would prefer if we defer this to the next release. In my opinion, it's better to coordinate with @inancgumus on similar refactors.

Reply:
@inancgumus do you have any plan in mind for it? Can we remove it or do we expect to add features where it is needed?

Reply:
It's better to keep it and add a TODO comment instead because here, we (the early-browser team) forgot to respect the options.

Reply:
https://github.com/grafana/k6/pull/4538